pkg/virtctl/root.go

```diff
@@ -55,22 +55,27 @@ func NewVirtctlCommand() *cobra.Command {
 		SilenceUsage:  true,
 		SilenceErrors: true,
 		Run: func(cmd *cobra.Command, args []string) {
-			fmt.Fprint(cmd.OutOrStderr(), cmd.UsageString())
+			cmd.Printf(cmd.UsageString())
 		},
 	}
 
+	//TODO: Add a ClientConfigFactory which allows substituting the KubeVirt client with a mock for unit testing
+	clientConfig := kubecli.DefaultClientConfig(rootCmd.PersistentFlags())
+	AddGlogFlags(rootCmd.PersistentFlags())
+	rootCmd.SetUsageTemplate(templates.MainUsageTemplate())
+
+	rootCmd.PersistentPreRunE = func(cmd *cobra.Command, args []string) error {
```

Comment:
nit: why don't you put this in the struct when you declare it? I think it's a tiny bit more readable.